.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 pontus.larsson_medsci.uu.se.key
 charon_credentials.yml
-statusdb_creds.yml
+statusdb_creds_prod.yml
```

Comment:
I wrote it so that {{ statusdb_creds }} is set to point to either statusdb_creds_prod.yml or statusdb_creds_stage.yml. I think that should work.